lib/Finance/Quote/Bourso.pm

```diff
@@ -125,194 +123,146 @@ sub bourso {
             # print $reply->content;
             $info{ $stocks, "success" } = 1;
 
-            my $tree = HTML::TreeBuilder->new_from_content( $reply->content );
-
-            # retrieve SYMBOL
-            my @symbolline = $tree->look_down( 'class', 'seoinline fv-isin ellipsis' );
-
-            unless (@symbolline) {
+            my $tp = HTML::TokeParser->new(\$reply->content );
```

Comment:
Maybe it would be more direct to search for the element with `data-faceplate-symbol = $stocks`?